{
  "UncaughtException": "In <Task> Error: Assertion failed: <Class Int>==<Class String>\nat assert_failed (assert.np:23)\nat <main> (assert_failed.np:3)"
}